### PR TITLE
Update how-to-restore-dropped-server.md (Typo "flexible server"→"Flex…

### PR DESCRIPTION
**Pull request by hyoshioka0128**
…ible Server")

https://github.com/MicrosoftDocs/azure-databases-docs/blob/main/articles/mysql/flexible-server/how-to-restore-dropped-server.md 
#PingMSFTDocs

### CODE REVIEW

**Comment by hyoshioka0128** [comment]
@VandhanaMehta - related #29

File: articles/mysql/flexible-server/how-to-restore-dropped-server.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ To restore a deleted Azure Database for MySQL flexible server instance, you need
 1. In the Activity Log, select **Add filter** as shown and set the following filters for the
 
     - **Subscription** = Your Subscription hosting the deleted server
-    - **Resource Type** = Azure Database for MySQL flexible servers (Microsoft.DBforMySQL/flexibleServers)
+    - **Resource Type** = Azure Database for MySQL Flexible Servers (Microsoft.DBforMySQL/flexibleServers)
```
Comment (ShannonLeavitt) [inline articles/mysql/flexible-server/how-to-restore-dropped-server.md:32]: Note that the instance in line 32 is plural, so it's not clear whether this refers to the product name (which should be capitalized) or to an instance, which might not need to be capitalized.

**Comment by ShannonLeavitt** [comment]
@VandhanaMehta

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by VandhanaMehta — APPROVED** [review]

File: articles/mysql/flexible-server/how-to-restore-dropped-server.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ To restore a deleted Azure Database for MySQL flexible server instance, you need
 Activity log by filtering for:
 
    - **Subscription** = Your Subscription
-   - **Resource Type** = Azure Database for MySQL flexible servers (Microsoft.DBforMySQL/flexibleServers)
+   - **Resource Type** = Azure Database for MySQL Flexible Servers (Microsoft.DBforMySQL/flexibleServers)
```
Comment (Jak-MS) [inline articles/mysql/flexible-server/how-to-restore-dropped-server.md:72]:
```suggestion
   - **Resource Type** = Azure Database for MySQL Flexible Server (Microsoft.DBforMySQL/flexibleServers)
```

**Comment by Jak-MS** [comment]
merged based on approval from @VandhanaMehta

**Pull request merged**